modules/seqexec/model/src/main/scala/seqexec/model/dhs.scala

```diff
@@ -3,9 +3,41 @@
 
 package seqexec.model
 
-object dhs {
+import cats.Eq
+import cats.Monoid
+import cats.implicits._
+import shapeless.tag.@@
+import shapeless.tag
+
+package dhs {
 
   // Common data types
-  type ImageFileId = String
+  trait ImageFileIdT
+  trait DataIdT
+
+}
+
+package object dhs {
+
+  type ImageFileId = String @@ ImageFileIdT
```

Comment:
There was a bug where ImageFileId and DataId were swapped, both `String`. Now we are tagging them